pkg/config/agentmanagement_test.go

```diff
@@ -132,6 +132,36 @@ func TestSleepTime(t *testing.T) {
 	assert.Equal(t, time.Second*15, st)
 }
 
+func TestJitterTime(t *testing.T) {
```

Comment:
Maybe is a good time to try [fuzz tests](https://go.dev/security/fuzz/)? We probably should also tests that given a min and a max, the function return values that are equally distributed among the possible return options